hypervisor/dm/vpci/vpci.c

```diff
@@ -64,19 +64,21 @@ static uint32_t pci_cfg_io_read(struct acrn_vm *vm, uint16_t addr, size_t bytes)
 				val |= PCI_CFG_ENABLE;
 			}
 		}
-	} else if (is_cfg_data(addr)) {
-		if (pi->cached_enable) {
-			uint16_t offset = addr - PCI_CONFIG_DATA;
+	} else {
+		if (is_cfg_data(addr)) {
+			if (pi->cached_enable) {
+				uint16_t offset = addr - PCI_CONFIG_DATA;
 
-			if ((vpci->ops != NULL) && (vpci->ops->cfgread != NULL)) {
-				vpci->ops->cfgread(vpci, pi->cached_bdf,
-					pi->cached_reg + offset, bytes, &val);
-			}
+				if ((vpci->ops != NULL) && (vpci->ops->cfgread != NULL)) {
+					vpci->ops->cfgread(vpci, pi->cached_bdf,
+						pi->cached_reg + offset, bytes, &val);
+				}
 
-			pci_cfg_clear_cache(pi);
+				pci_cfg_clear_cache(pi);
+			}
+		} else {
+			val = 0xFFFFFFFFU;
 		}
```

Comment:
above change is a pure coding style change, right?

Reply:
Yes, only added the missing "{}" to fix the "'No brackets to then/else'" violation.